uat/custom-components/client-mosquitto-c/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:22.04 AS builder
```

Comment:
please use an image from ECR public gallery and not dockerhub: https://gallery.ecr.aws/

Reply:
When replaced to public.ecr.aws/lts/ubuntu:22.04_stable I got an error in git clone:
```
Cloning into 'grpc-src'...
fatal: unable to access 'https://github.com/grpc/grpc/': server certificate verification failed. CAfile: none CRLfile: none
```
Looks like that image lack the trusted CA list.
Do we need install CAs manually ?

Reply:
Try using amazonlinux.

Sounds like that image may have outdated CA certs perhaps.

Reply:
Image ubuntu:22.04_stable  completely missing root CAs  I thing due to package was not installed at all. Solution is reinstall ca-certificates package. Now testing that solution, build takes a time due to gRPC + protobuf + all requirements added as submodules are building.

amazonlinux2 is centOS based distro, it potentially can cause a problem because github uses ubuntu 22.04 on default runners

Reply:
Resolved in 8505bb92b8d5fabed2f7ba05324d44540b95af8d